modules/conversion-center/module.php

```diff
@@ -144,10 +168,58 @@ public function __construct() {
 		}, 100 );
 	}
 
+	public function handle_click_tracking() {
+		$data = filter_input_array( INPUT_POST, [
+			'clicks' => [
+				'filter' => FILTER_VALIDATE_INT,
+				'flags' => FILTER_REQUIRE_ARRAY,
+			],
+			'_nonce' => FILTER_UNSAFE_RAW,
+		] );
+
+		if ( ! wp_verify_nonce( $data['_nonce'], 'elementor-pro-frontend' ) ) {
+			wp_send_json_error( [ 'message' => 'Invalid nonce' ] );
+		}
+
+		if ( ! check_ajax_referer( 'elementor-pro-frontend', '_nonce', false ) ) {
+			wp_send_json_error( [ 'message' => 'Invalid referrer' ] );
+		}
+
+		$posts_to_update = [];
+
+		foreach ( $data['clicks'] as $post_id ) {
+			if ( ! isset( $posts_to_update[ $post_id ] ) ) {
+				$starting_clicks = (int) get_post_meta( $post_id, '_elementor_click_tracking', true );
+				$posts_to_update[ $post_id ] = $starting_clicks ? $starting_clicks : 0;
+			}
+			$posts_to_update[ $post_id ] ++;
+		}
+
+		foreach ( $posts_to_update as $post_id => $clicks ) {
+			update_post_meta( $post_id, '_elementor_click_tracking', $clicks );
+		}
+
+		wp_send_json_success( [ 'nonce' => wp_create_nonce( 'elementor-pro-frontend' ) ] );
+	}
+
 	public function set_admin_columns_content( $column_name, $post_id ) {
 		$document = Plugin::$instance->documents->get( $post_id );
 
 		$document->admin_columns_content( $column_name );
+		switch ( $column_name ) {
+			case 'click_tracking':
+				$click_tracking = get_post_meta( $post_id, '_elementor_click_tracking', true );
```

Comment:
```suggestion
				$click_tracking = get_post_meta( $post_id, SOME_CONST_NAME, true );
```
Might be good to use a const for this?

Reply:
@louiswol94 yes, good idea, added!